docs/api/endorsement-provisioning/endorsement-provisioning.yaml

```diff
@@ -0,0 +1,71 @@
+openapi: '3.0.0'
+
+info:
+  title: attestation endorsement provisioning
+  description: For further details, see datatracker.ietf.org/doc/draft-birkholz-corim
+  version: '1.0.0alpha'
+servers:
+  - url: 'https://veraison.example/endorsement-provisioning/v1'
+paths:
+  /submit:
+    post:
+      description: >
+        Submit a single CoRIM or a batch of CoRIM payloads using this API.
+      requestBody:
+        required: true
+        content:
+          application/rim+cbor:
+            schema:
+              $ref: './schemas/components.yaml#/components/schemas/Submission'  
```

Comment:
I am not sure we need a schema here: it's just binary data and a content-type

Reply:
No it is good to keep it like that for consistency!

Reply:
Need to incline the schema!